manifests/a/Autodesk/DesktopApp/8.4.0.50/Autodesk.DesktopApp.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Autodesk.DesktopApp
+PackageVersion: 8.4.0.50
+PackageLocale: en-US
+Publisher: Autodesk Inc.
```

```suggestion
Publisher: Autodesk
```
This must match Control Panel.